### ISSUE TITLE
Setup Sphinx extension infrastructure

### ISSUE THREAD

**Issue by Obarrie** [csctd/sphinx_pechakucha#2]
- https://www.sphinx-doc.org/en/master/development/tutorials/todo.html

**Comment by brownsarahm** [comment]
how is this going?

**Comment by Obarrie** [comment]
Isaac and I are working on it, we will have something for you by Sunday.

**Comment by isaacgbaba** [comment]
Some quick questions on setting everything up properly we went ahead and installed sphinx using the pip install spinx  and got a basic overview of what is is using YouTube and the spinx site 

for our next step how do we go about making the extension to do what we need we are kind of stuck and need a little push

**Comment by brownsarahm** [comment]
So the other things to do is to make sure you are working with this and the [test case](https://github.com/csctd/pechakuchaTest) are both downloaded and then see if you can use the "todo" directive as setup here from the tutorial in your test case.  Once you get used to using that then the next steps will be more clear.

**Comment by Obarrie** [comment]
- I am a little confused about this line "So the other things to do is to make sure you are working with this"

-Also for this line " then see if you can use the "todo" directive as setup here from the tutorial in your test case." 
 Do you mean the this code "https://github.com/csctd/sphinx_pechakucha/blob/main/_ext%20folder/todo.py"

**Comment by brownsarahm** [comment]
yeah, you have hat "todo" directive from the tutorial setup, but the next thing to do is to make sure that it is fully setup and your test case is also setup right (as in this extension is used correctly by the test repo)

**Comment by Obarrie** [comment]
Oh okay so is this how the todo and todolist work?

https://github.com/csctd/pechakuchaTest/blob/main/index.rst
https://github.com/csctd/pechakuchaTest/blob/main/somefile.rst

we commented out the other code from the conf.py file and just included the todo extension. 

I am just curious is that how it is going to work and are these the steps?
1. write the code for the pechakucha in vscode
2. include it in a file in the GitHub
3. include that file in conf.py
4. Add said file to the extensions 
5. and whenever they run "..pechakucha::" it will just do it for them?

**Comment by Obarrie** [comment]
I just made a pull request under the test repo so you can take a look when you can please?
